dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/custom/EuTaxonomyAlignedActivitiesComponent.kt

```diff
@@ -0,0 +1,76 @@
+package org.dataland.frameworktoolbox.frameworks.eutaxonomynonfinancials.custom
+
+import org.apache.commons.text.StringEscapeUtils
+import org.dataland.frameworktoolbox.intermediate.FieldNodeParent
+import org.dataland.frameworktoolbox.intermediate.components.ComponentBase
+import org.dataland.frameworktoolbox.intermediate.components.addStandardCellWithValueGetterFactory
+import org.dataland.frameworktoolbox.intermediate.components.requireDocumentSupportIn
+import org.dataland.frameworktoolbox.intermediate.datapoints.NoDocumentSupport
+import org.dataland.frameworktoolbox.specific.datamodel.TypeReference
+import org.dataland.frameworktoolbox.specific.datamodel.elements.DataClassBuilder
+import org.dataland.frameworktoolbox.specific.fixturegenerator.elements.FixtureSectionBuilder
+import org.dataland.frameworktoolbox.specific.uploadconfig.elements.UploadCategoryBuilder
+import org.dataland.frameworktoolbox.specific.viewconfig.elements.SectionConfigBuilder
+import org.dataland.frameworktoolbox.specific.viewconfig.elements.getTypescriptFieldAccessor
+import org.dataland.frameworktoolbox.specific.viewconfig.functional.FrameworkDisplayValueLambda
+import org.dataland.frameworktoolbox.utils.typescript.TypeScriptImport
+
+/**
+ * Represents the EuTaxonomy-Specific "EuTaxonomyAlignedActivities" component
+ */
+class EuTaxonomyAlignedActivitiesComponent(
+    identifier: String,
+    parent: FieldNodeParent,
+) : ComponentBase(
+    identifier, parent,
+) {
+    override fun generateDefaultViewConfig(sectionConfigBuilder: SectionConfigBuilder) {
+        sectionConfigBuilder.addStandardCellWithValueGetterFactory(
+            this,
+            FrameworkDisplayValueLambda(
+                "formatEuTaxonomyNonFinancialsAlignedActivitiesDataForTable(" +
+                    "${getTypescriptFieldAccessor(true)}, \"${
+                        StringEscapeUtils.escapeEcmaScript(
+                            label,
+                        )
+                    }\")",
+                setOf(
+                    TypeScriptImport(
+                        "formatEuTaxonomyNonFinancialsAlignedActivitiesDataForTable",
+                        "@/components/resources/dataTable/conversion/" +
+                            "EuTaxonomyNonFinancialsAlignedActivitiesDataGetterFactory",
+                    ),
+                ),
+            ),
+        )
+    }
+
+    override fun generateDefaultDataModel(dataClassBuilder: DataClassBuilder) {
+        dataClassBuilder.addProperty(
+            this.identifier,
+            TypeReference(
+                "kotlin.collections.MutableList",
+                true,
```

Comment:
Why is this not nullable instead of true?

Reply:
Du hast recht.
Angepasst.